e2e/rbd.go

```diff
@@ -2595,10 +2662,16 @@ var _ = Describe("RBD", func() {
 					e2elog.Failf("failed to check csi journal in pool: %v", err)
 				}
 
+				err = deleteJournalInfoInPool(f, pvc, "replicapool")
+				if err != nil {
+					e2elog.Failf(" failed to delete omap data: %v", err)
+				}
```

Comment:
@pkalever can you explain more about this one? why this is required in E2E.

Reply:
@Madhu-1 The omap tests had caught there are 2 omap leaks, at each place I used `deleteJournalInfoInPool` there is 1 leak. The description of `deleteJournalInfoInPool` says they clean the omaps.

Is there any other way to clean the omaps here?

Reply:
We should identify why omap is getting leaked in actual code instead of cleaning it in E2E. if we don't do that we might have leaks in the real environment isn't it?

Reply:
I thought the e2e is not cleaning it properly. I didn't understand the e2e test much. Can you please check on it once?

Reply:
we are just validating the omap created by actual cephcsi deployment for each pvc and snapshots in E2E.
it's not a problem with the E2E, it should be a problem with cephcsi itself. can you please open an issue to track this one? once we fix this one we need to remove the workaround.

Reply:
Got it, sure Madhu!

Reply:
Also I make it clear in the commit description that this is a workaround fix only.

Reply:
```suggestion
				err = deleteJournalInfoInPool(f, pvc, "replicapool")
				if err != nil {
					e2elog.Failf(" failed to delete omap data: %v", err)
				}
```
@pkalever even if this is a workaround, should it not be after `deletePVCAndApp()` ?

Reply:
Yes correct I missed this one, @pkalever can you please check.